invenio_app_ils/ui/backoffice/src/common/api.js

```diff
@@ -27,6 +27,10 @@ export const $axios = axios.create({
   withCredentials: true,
 });
 
+export const fetchList = url => {
```

Comment:
I agree on this one too, I am in the process of separating the api according to the models. The `commons` directory is not there to be used as a bucket to put everything rather than a place for all common functionality, shared components, services etc. For the `$` convention it is indeed a jQuery left over, as an angular convention for private variables. We can prefix it with `_` which is the Underscore convention, in general I don't like `axions_inst` or postfix `i` for interfaces etc. JavaScript has a very [wide acceptance](https://mathiasbynens.be/notes/javascript-identifiers) for what it accepts as valid characters for variable names but I guess `ಠ_ಠaxios` beats the purpose of naming conventions.